lib/cmdledit.g

```diff
@@ -751,10 +751,108 @@ GAPInfo.CommandLineEditFunctions.Functions.(INT_CHAR(' ')) :=
                 GAPInfo.CommandLineEditFunctions.Functions.SpaceDeletePrompt;
 BindKeysToGAPHandler(" ");
 
+
+BindGlobal("STANDARD_EXTENDERS", rec(
+  caseSensitive := function(cand, word)
+      local i, j, c;
+      i := Length(word);
+      while true do
+        if i = Length(cand[1]) then
+          break;
+        fi;
+        c := cand[1][i+1];
+        for j in [2..Length(cand)] do
+          if Length(cand[j]) > i and cand[j][i+1] = c then
+            j := j+1;
```

Comment:
Ah OK, I see now that this was the original code, and you simply copied and then edited it.

I still think that it was / is needlessly arcane :-)